src/main/java/com/minecolonies/entity/EntityCitizen.java

```diff
@@ -624,6 +630,16 @@ public void onLivingUpdate()
             pickupItems();
             cleanupChatMessages();
             updateColonyServer();
+            if(worldObj.isDaytime() && !worldObj.isRaining())
+            {
+                SoundUtils.playRandomSound(worldObj, this);
+            }
+            else if(worldObj.isRaining() && 1 >=rand.nextInt(RANT_ABOUT_WEATHER_CHANCE))
+            {
+                //todo add sounds of other workers as well.
+                final SoundEvent badWeather = isFemale() ? FishermanSounds.Female.badWeather : FishermanSounds.Male.badWeather;
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)